bot/config.py

```diff
@@ -0,0 +1,8 @@
+'''
+This file gets token from txt
+'''
+
+TOKEN = None
+
+with open("token.txt", encoding="utf-8") as t:
```

Comment:
I thought we discussed not reading it from the file but have it in .py.sample as a constant and have this .py.sample under .gitignore.